src/MLOps.NET/Storage/Database/MLOpsDbContext.cs

```diff
@@ -39,9 +38,9 @@ public async Task SaveChangesAsync()
         /// Ensured that the database is created
```

Comment:
Should this comment be updated to migrate rather than creation? Or be clearer that it is using the migrate method rather than ensure...

Reply:
Great point, I'll make it a bit clearer depending on if it's a relational or non-relational DB provider